src/main/java/com/google/sps/data/AQDataPoint.java

```diff
@@ -4,10 +4,31 @@
  * This class represents the unified data format for AQ data passed to the client.
  */
 public class AQDataPoint {
-    private int lat;
-    private int lng;
-    private double aqi;
-    private String siteName;
+  private double lat;
+  private double lng;
+  private double aqi;
+  private String siteName;
 
-    public AQDataPoint() {}
+  public AQDataPoint() {}
+
+  public void setLat(double lat) {
```

Comment:
Why does it need setters? Wasn't there some discussion about just having the variables public for PDS's

Reply:
Yep, AQDataPoint should be a public since it will be the common format that will be returned from the adaptor